tests/test_examples/conftest.py

```diff
@@ -0,0 +1,48 @@
+import os
+
+import pytest
+from dotenv import load_dotenv
+from pydantic import SecretStr
+
+from odp_sdk.auth import AzureTokenProvider
+from odp_sdk.client import OdpClient
+
+__all__ = ["odp_client"]
+
+
+@pytest.fixture(scope="session")
+def dotenv() -> None:
+    load_dotenv()
+
+
+@pytest.fixture(scope="session")
+def token_provider(dotenv) -> AzureTokenProvider:
+    return AzureTokenProvider(
+        authority=os.getenv(
+            "ODCAT_AUTH_AUTHORITY",
+            "https://oceandataplatform.b2clogin.com/755f6e58-74f0-4a07-a599-f7479b9669ab/v2.0/",
+        ),
+        client_id=SecretStr(os.getenv("ODCAT_AUTH_CLIENT_ID")),
+        client_secret=SecretStr(os.getenv("ODCAT_AUTH_CLIENT_SECRET")),
+        audience=os.getenv("ODCAT_AUTH_AUDIENCE", "a2e4df44-ed57-4673-8824-548256b92543"),
+        tenant_id=os.getenv("ODCAT_AUTH_TENANT_ID", "755f6e58-74f0-4a07-a599-f7479b9669ab"),
```

Comment:
is this safe to be hardcoded here?

Reply:
I think so, it's the same as here: https://github.com/C4IROcean/app-odcat/blob/fd8ba35462bded4ed84d23d8e8ee719073bdedb9/tests/test_api/fixtures/auth.py#L20